src/test/java/org/fogbowcloud/saps/engine/core/archiver/storage/NfsPermanentStorageTest.java

```diff
@@ -0,0 +1,105 @@
+package org.fogbowcloud.saps.engine.core.archiver.storage;
+
+import java.io.File;
+import java.sql.Timestamp;
+import java.util.Date;
+import java.util.Objects;
+import java.util.Properties;
+import org.apache.commons.io.FileUtils;
+import org.fogbowcloud.saps.engine.core.archiver.storage.exceptions.PermanentStorageException;
+import org.fogbowcloud.saps.engine.core.model.SapsImage;
+import org.fogbowcloud.saps.engine.core.model.enums.ImageTaskState;
+import org.fogbowcloud.saps.engine.utils.SapsPropertiesConstants;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class NfsPermanentStorageTest {
```

Comment:
Create tests for archiving and deleting a failed task. This is related to the altering path of the constant `TASK_DIR_PATTERN` according to the mode of operation of the Archiver component (debug mode is active).

Reply:
Done.